modes/tmtv/src/initToolGroups.js

```diff
@@ -30,6 +30,7 @@ function initToolGroups(toolNames, Enums, ToolGroupService, commandsManager) {
       { toolName: toolNames.DragProbe },
       { toolName: toolNames.EllipticalROI },
       { toolName: toolNames.RectangleROI },
+      { toolName: toolNames.PlanarFreehandROI },
```

Comment:
tmtv mode does not need freehand tool since there is no measurement tracking/export etc

Reply:
ok